### PR TITLE
Add medical HUD icon if the body has died recently enough to be defibbed

### PR DESCRIPTION
**Pull request by Mothblocks**
[Changelogs]: 

:cl:
imageadd: Medical HUDs will now show an icon if the body has died recently enough to be defibbed
/:cl:

[why]: Health analyzers already will tell you this, so this just makes it easier. I read this on the "Minor Suggestions" thread. I'm no spriter, so feel free to tell me if my icon sucks. I had to add an "addtimer" to the death proc because med_hud_set_status isn't called anymore after death.

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
I was thinking that the background color was bad for what it means. I'll try a pulsing red background and send a video of it to see if I can make it any better.

**Comment by XDTM** [comment]
i suggest you try the light blue on black first, it's closer to the death icon while red might look more like deep crit now that i think about it

**Comment by Jared-Fogle** [comment]
As in the line is light blue and the background is black?

**Comment by XDTM** [comment]
Exactly

By the way, to show an animation you just need to export it as a gif in byond

**Comment by Jared-Fogle** [comment]
defib
@XDTM How about this?

**Comment by XDTM** [comment]
I agree, looks good

**Comment by SpaceManiac** [comment]
What's the deal with there being `timeofdeath` AND `tod` vars?

Which of the two does fake death set? Is someone faking death going to appear defibbable forever?

**Comment by XDTM** [comment]
Fakedeath sets tod to the moment it is applied, so it looks like someone just died

**Comment by SpaceManiac** [comment]
Based on a quick read fakedeath does not set `timeofdeath` so a fake death will never look defibbable

**Comment by Jared-Fogle** [comment]
@SpaceManiac Then it'll never look defibbable on a medical analyzer either, as that's where I copied the code from.

File: code/game/data_huds.dm [hunk 1]
```diff
@@ -175,6 +175,11 @@
 	if(has_trait(TRAIT_XENO_HOST))
 		holder.icon_state = "hudxeno"
 	else if(stat == DEAD || (has_trait(TRAIT_FAKEDEATH)))
+		if(tod)
```
Comment (XDTM) [inline code/game/data_huds.dm]: tod

**Comment by ExcessiveUseOfCobblestone** [comment]
If you're not actually dead will it let you defib anyways? I would imagine the safeties will let you meta it anywho.

**Comment by XDTM** [comment]
Yeah but i'm sure there's plenty of methods anyway that don't require a defib

**Review by SpaceManiac — APPROVED** [review]
works in testing

**Pull request merged**